src/hall_of_fame.c

```diff
@@ -1,4 +1,1678 @@
+    {326,   244,    56,     64},
+    {-86,   244,    184,    64}
+};
+
+static const struct OamData sOamData_85E53FC =
+{
+    .y = 0,
+    .affineMode = 0,
+    .objMode = 0,
+    .mosaic = 0,
+    .bpp = 0,
+    .shape = 0,
+    .x = 0,
+    .matrixNum = 0,
+    .size = 0,
+    .tileNum = 0,
+    .priority = 0,
+    .paletteNum = 0,
+    .affineParam = 0,
+};
+
+static const union AnimCmd sSpriteAnim_85E5404[] =
+{
+    ANIMCMD_FRAME(0, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E540C[] =
+{
+    ANIMCMD_FRAME(1, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5414[] =
+{
+    ANIMCMD_FRAME(2, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E541C[] =
+{
+    ANIMCMD_FRAME(3, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5424[] =
+{
+    ANIMCMD_FRAME(4, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E542C[] =
+{
+    ANIMCMD_FRAME(5, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5434[] =
+{
+    ANIMCMD_FRAME(6, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E543C[] =
+{
+    ANIMCMD_FRAME(7, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5444[] =
+{
+    ANIMCMD_FRAME(8, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E544C[] =
+{
+    ANIMCMD_FRAME(9, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5454[] =
+{
+    ANIMCMD_FRAME(10, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E545C[] =
+{
+    ANIMCMD_FRAME(11, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5464[] =
+{
+    ANIMCMD_FRAME(12, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E546C[] =
+{
+    ANIMCMD_FRAME(13, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5474[] =
+{
+    ANIMCMD_FRAME(14, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E547C[] =
+{
+    ANIMCMD_FRAME(15, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_85E5484[] =
+{
+    ANIMCMD_FRAME(16, 30),
+    ANIMCMD_END
+};
+
+static const union AnimCmd * const sSpriteAnimTable_85E548C[] =
+{
+    sSpriteAnim_85E5404, sSpriteAnim_85E540C, sSpriteAnim_85E5414, sSpriteAnim_85E541C,
+    sSpriteAnim_85E5424, sSpriteAnim_85E542C, sSpriteAnim_85E5434, sSpriteAnim_85E543C,
+    sSpriteAnim_85E5444, sSpriteAnim_85E544C, sSpriteAnim_85E5454, sSpriteAnim_85E545C,
+    sSpriteAnim_85E5464, sSpriteAnim_85E546C, sSpriteAnim_85E5474, sSpriteAnim_85E547C,
+    sSpriteAnim_85E5484
+};
+
+static const struct SpriteTemplate sSpriteTemplate_85E54D0 =
+{
+    .tileTag = 1001,
+    .paletteTag = 1001,
+    .oam = &sOamData_85E53FC,
+    .anims = sSpriteAnimTable_85E548C,
+    .images = NULL,
+    .affineAnims = gDummySpriteAffineAnimTable,
+    .callback = sub_81751A4
+};
+
+// todo: dump actual graphics
+static const u32 sHallOfFame_Pal[] =
```

Comment:
ew 2 electric boogaloo

Reply:
im the decomp guy, not the dump grahpics guy

Reply:
When you decomp, please extract as well.